packages/manager/modules/iam/src/components/createPolicy/translations/Messages_fr_FR.json

```diff
@@ -15,7 +15,6 @@
   "iam_create_policy_form_name_heading": "Nom de la politique",
   "iam_create_policy_form_resource_groups_create": "Créer un groupe de ressources",
   "iam_create_policy_form_resource_groups_heading": "Groupes de ressources",
-  "iam_create_policy_form_resource_groups_info": "Vous n'avez pas encore créé de groupe de resources. [TODO]complete this message[/TODO]",
```

Comment:
**question:** Is this key used anywhere?

Reply:
No, the UI that used it was removed in a previous PR, this key should have been removed along with it